internal/pkg/epinject/ociregistry/server.go

```diff
@@ -265,19 +265,8 @@ func (s *Server) FetchBlob(name string, id string) ([]byte, error) {
 }
 
 func (s *Server) uploadEntrypoint() error {
-	found, err := s.probeBlob(s.entrypointRepo, s.entrypointId)
-	if err != nil {
-		return err
-	}
-
-	if found {
-		s.Logger.Debug("entrypoint layer already exists, not reuploading")
-		return nil
-	}
-
```

Comment:
We can put this back in, it will make things faster and wasn't causing an issue.

Reply:
Restored in 4480ed51f